### PR TITLE
Add enableCommand method to RestrictedEditingModeEditing.

### PR DESCRIPTION
**Pull request by jodator**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Add enableCommand method to RestrictedEditingModeEditing. Closes ckeditor/ckeditor5#6041. Closes ckeditor/ckeditor5#6011.

---

### Additional information

* This PR add only a `RestrictedEditingModeEditing.enableCommand()` method.

### CODE REVIEW

**Comment by jodator** [comment]
@Reinmar I was considering also an `allowCommand()` method name (to make this coherent with configuration option) but I've used method name from track changes which does the same.

File: src/restrictededitingmodeediting.js [hunk 1]
```diff
@@ -92,6 +95,28 @@ export default class RestrictedEditingModeEditing extends Plugin {
 		} );
 	}
 
+	/**
+	 * Enables command with given `commandName` in restricted editing mode.
```
Comment (Reinmar) [inline src/restrictededitingmodeediting.js]:
```suggestion
	 * Makes the given command enabled when selection is located in an editable region.
```

File: src/restrictededitingmodeediting.js [hunk 2]
```diff
@@ -92,6 +95,28 @@ export default class RestrictedEditingModeEditing extends Plugin {
...
+	 *
+	 * @param {String} commandName Name of the command to enable.
+	 */
+	enableCommand( commandName ) {
+		const command = this.editor.commands.get( commandName );
+
+		if ( !command ) {
```
Comment (Reinmar) [inline src/restrictededitingmodeediting.js]: Seems unnecessary. If it's going to fail, it's going to fail in the very next line so it's super easy to debug.

**Comment by Reinmar** [comment]
> @Reinmar I was considering also an `allowCommand()` method name (to make this coherent with configuration option) but I've used method name from track changes which does the same.

It's ok as long as both things (config and this method) are cross-linked.

File: src/restrictededitingmodeediting.js (hunk 2 above)
Comment (jodator) [reply src/restrictededitingmodeediting.js]: Done.

**Comment by jodator** [comment]
@Reinmar merged as this PR was needed by @scofalik - feel free to do a post merge review if needed :) but the requested issues were already adressed.

**Pull request merged**